src/Tests/Microsoft.NET.TestFramework/ToolsetInfo.cs

```diff
@@ -423,5 +423,16 @@ public static string GetNewtonsoftJsonPackageVersion()
             return _NewtonsoftJsonPackageVersion.Value;
         }
 
+        private static readonly Lazy<string> _SystemDataSqlClientPackageVersion = new(() =>
```

Comment:
```suggestion
                private static readonly Lazy<string> _SystemDataSqlClientPackageVersion = new Lazy<string>(() => GetPackageVersion("SystemDataSqlClientPackageVersion"));

        private static string GetPackageVersion(string key)
        {
            Assembly assembly = Assembly.GetExecutingAssembly();
            return assembly.GetCustomAttributes(true)
                .OfType<AssemblyMetadataAttribute>()
                .FirstOrDefault(a => a.Key == key)?.Value;
        }
```
May I suggest the above change so that the logic can be reused for the two package versions and so we can potential reuse it further in the future (if we identify more packages that we use in tests and want to centralize the versions for)